### ISSUE TITLE
A-extension tests

### ISSUE THREAD

**Issue by algrobman** [riscv-non-isa/riscv-arch-test#293]
I couldn't find AMO/LR/SC tests, where are they?

**Comment by allenjbaum** [comment]
The really important aspect of the A-extension is how it behaves when there
are concurrent access, and we don't have a way to make current accesses
with Sail and the framework (yet).
It is coming,. Having said thatL we can test basic non-concurrent
functionality. Would yo ulike to volunteer to write tests?

**Comment by algrobman** [comment]
@allenjbaum , I completely agree with your assessment, but at least level 1 tests for single core are still very useful. 
I would like to help, but still can't make riscof installed ...

**Comment by allenjbaum** [comment]
Have you asked your IT admin for help? It you don't have the permissions,
it is the only way to get them. If they have questions, refer them to me.

**Comment by UmerShahidengr** [comment]
AMO tests have been merged, LR/SC tests are still in review, will be available soon

**Comment by allenjbaum** [comment]
There is a coverage hole; I believe that the .w versions simply access the current signature doubleword (for RV64) and therefore never try to access just the upper half, and we need to add those tests as well (as well as coverpoints) for each opcode.

**Comment by jordancarlin** [comment]
AMO and LR/SC tests have both been added in ACT4.
